src/olympia/legacy_api/views.py

```diff
@@ -388,13 +385,15 @@ def process_request(self, query, addon_type='ALL', limit=10,
             .filter_query_string(query)
             [:limit])
 
+        results = []
+
         total = qs.count()
 
-        results = []
```

Comment:
Irrelevant, I used this for debugging. I'll remove this and a few other unrelated changes.